Mathlib/LinearAlgebra/AffineSpace/AffineMap.lean

```diff
@@ -853,7 +831,7 @@ theorem homothety_apply_same (c : P1) (r : k) : homothety c r c = c :=
 
 theorem homothety_mul_apply (c : P1) (r₁ r₂ : k) (p : P1) :
     homothety c (r₁ * r₂) p = homothety c r₁ (homothety c r₂ p) := by
-  simp [homothety_apply, mul_smul]
+  simp only [homothety_apply, mul_smul, vadd_vsub]
```

Comment:
This proof was ported without modification the first time, but forever reason needs more help after this re-port